source/kernel/Makefile

```
@@ -84,326 +119,47 @@ endef
+#
+# Sub-makes within a single variant are incremental, but switching
+# variants between sub-makes forces a full reassembly (intrinsic to the
+# single-stamp mechanism). The default variant is built last so the
+# on-disk intermediate file (nextor_base.dat) is left as the
+# non-inverted, undoc kernel that downstream consumers (source/drivers)
+# pick up by default.
+#
+# Each sub-make is prefixed with `-` and passes --keep-going so a
+# failure in one variant does not stop the other five from running.
+.PHONY: everything
+everything:
```

checkmake (0.3.2)

[warning] Target body for "everything" exceeds allowed length of 5 lines (6).

(maxbodylength)